test/AutoRemediateS3-002.test.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
+
+const source = require('../functions/AutoRemediateS3-002');
+
+//const config = require('./config');
+const AWS = require("aws-sdk");
+
+const CCRuleCode = 'S3-002';
+const CCRuleName = 'BucketPublicReadAcpAccess';
+const allUsersURI = 'http://acs.amazonaws.com/groups/global/AllUsers';
+const readAcpPermission = "READ_ACP";
+const aclSkeleton = '{"Owner":"", "Grants":[]}'; // skeleton for new permission grants
```

Comment:
In Javascript, if you want a JSON object, you can create one directly - you don't have to parse from JSON string.

```
const aclSkeleton = { Owner: "", Grants: [] }
```